docs/apis-tools/orchestration-cluster-api-rest/orchestration-cluster-api-rest-overview.md

```diff
@@ -113,56 +151,54 @@ For IDs or similar short 2- or 3-letter words or acronyms, Camunda only capitali
-| 401        | Unauthorized. The client is not authenticated yet. The client should try again with a modified authorization header.                                                       |
-| 403        | Forbidden. The client has incorrect or insufficient permissions for the request.                                                                                           |
-| 404        | Not found                                                                                                                                                                  |
-| 409        | Conflict. The request is trying to modify a resource that is currently not in the right state.                                                                             |
-| 412        | Precondition failed. The client should check the cluster status.                                                                                                           |
-| 429        | Rate limit exceeded. The client exceeds a defined limit of requests, for example, Zeebe signaling backpressure due to more requests than the broker can currently process. |
-| 500        | Internal server error. Generic error that contains further description in the problem detail.                                                                              |
+| Error code | Meaning                                                                                                     |
+| ---------- | ----------------------------------------------------------------------------------------------------------- |
+| 200        | OK                                                                                                          |
+| 204        | No content                                                                                                  |
+| 400        | Bad request. Generic error with further description in the problem detail.                                  |
+| 401        | Unauthorized. The client is not authenticated. Retry with a modified authorization header.                  |
+| 403        | Forbidden. The client has insufficient permissions for the request.                                         |
+| 404        | Not found                                                                                                   |
+| 409        | Conflict. The request attempts to modify a resource that is not in the correct state.                       |
+| 412        | Precondition failed. The client should check the cluster status.                                            |
+| 429        | Rate limit exceeded. The client exceeds a defined request limit, for example, Zeebe signaling backpressure. |
+| 500        | Internal server error. Generic error with further description in the problem detail.                        |
```

Comment:
```suggestion
| 500        | Internal server error. Generic error with further description in the problem detail.                        |
| 503        | The service is currently unavailable. This may happen when the system signals backpressure to prevent the server's compute resources from being exhausted, avoiding more severe failures. Clients are recommended to eventually retry those requests after a backoff period. You can learn more about the backpressure mechanism [here](../../components/zeebe/technical-concepts/internal-processing.md) |
```